FLAnimatedImageDemo/FLAnimatedImage/FLAnimatedImageView.m

```diff
@@ -259,6 +259,10 @@ - (void)displayDidRefresh:(CADisplayLink *)displayLink
             if (self.currentFrameIndex >= self.animatedImage.frameCount) {
                 // If we've looped the number of times that this animated image describes, stop looping.
                 self.loopCountdown--;
+                if (self.loopCompletionBlock) {
+                    self.loopCompletionBlock(self, self.loopCountdown);
```

Comment:
I don't see how. But I'm not sure of the benefits to pass itself as a parameter anyway. I added it in case any other dev that would use this feature would need it.